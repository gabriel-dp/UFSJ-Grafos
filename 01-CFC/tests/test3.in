A B
A C
B D
B F
C D
D C
E A 
E B
F E
G E
G H
H F
H G