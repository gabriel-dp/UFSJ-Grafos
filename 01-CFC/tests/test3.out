{ A B E F }
{ C D }
{ G H }
